portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionsSamples.testcase

```diff
@@ -190,9 +190,22 @@ definition {
 
 			echo(${springBootLogFileContent});
 
+			var found = false;
```

Comment:
I think this has to be a string `"false"`, then the later comparison will look like:
```
if (${found} != "true") {
}
```